src/databricks/labs/ucx/azure/access.py

```diff
@@ -317,6 +317,8 @@ def create_access_connectors_for_storage_accounts(self) -> list[tuple[AccessConn
             list[AccessConnector, str] : The access connectors with a storage url to which it has access.
         """
         used_storage_accounts = self._get_storage_accounts()
+        if len(list(self._ws.storage_credentials.list())) > 200:
```

Comment:
Storage credentials are not yet made, so it should be:

```suggestion
        if len(list(used_storage_accounts)) > 200:
```

Note, we need logic that handles the already existing storage credentials to be excluded